Cargo.toml

```diff
@@ -48,7 +49,8 @@ v2_58 = ["v2_56", "glib-sys/v2_58", "gobject-sys/v2_58"]
 v2_60 = ["v2_58", "glib-sys/v2_60"]
 v2_62 = ["v2_60", "glib-sys/v2_62", "gobject-sys/v2_62"]
 v2_64 = ["v2_62", "glib-sys/v2_64"]
-dox = ["glib-sys/dox", "gobject-sys/dox"]
+bridged_logging = ["v2_46", "log"]
```

Comment:
Which part needs GLib 2.46?

Reply:
Ooops. My eyes had been tricked by 

```rust
mod log;
#[cfg(any(feature = "v2_46", feature = "dox"))]
pub use log::log_set_handler;
```

You are right, there's no need for v2_46 here.